dataprofiler/reports/graphs.py

```diff
@@ -19,7 +21,7 @@
 
 
 @utils.require_module(["matplotlib", "seaborn"])
-def plot_histograms(profiler, column_names=None, column_inds=None):
+def plot_histograms(profiler: StructuredProfiler, column_names: List[Union[int,str]] = None, column_inds: List[int] = None) -> plt.figure:
```

Comment:
instead of `plt.figure` I'd recommend `matplotlib.pyplot.figure` if possible for specificity and so it isn't requiring a particular way of importing matplotlib.

this might also be an opportunity for `from __futures__ import annotations`